electronic_store/data/product_product.xml

```diff
@@ -0,0 +1,426 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="product_product_1048" model="product.product">
+        <field name="name">1 Year Extended Warranty</field>
+        <field name="detailed_type">service</field>
+        <field name="list_price">100.0</field>
+        <field name="image_1920" type="base64" file="electronic_store/static/src/binary/product_template/1049-image_1920"/>
+    </record>
```

Comment:
There is a weird offset between the id (here 1048) and the image name (here 1049) throughout the whole file. Could you fix that?